compose/docker-compose.yml

```diff
@@ -1,13 +1,19 @@
 version: "2.0"
 services:
     lowPowerCollectd:
-         image: jstubbs/low_power_collectd
+         image: awbarnes/low_power_collectd:latest
```

Comment:
You can publish this to our docker registry too if you call it like `docker.chameleoncloud.org/low_power_collectd:latest`.